### ISSUE TITLE
Difficult to import 'angular-module-factory' as ES6 module.

### ISSUE THREAD

**Issue by cosemansp** [Swimlane/angular-model-factory#65]
I am trying to use the angular-model-factory in combination with bower and ES6 module import.
Due the fact that there is no main file in the package.json file it's difficult to import 'angular-model-import' directly.

    import modelFactory from 'angular-model-factory';   // fails due to missing main file in package.json
    import modelFactory from 'angular-model-factory/dist/angular-model-factory';  // this is working.

If your export the module name (module.exports = 'moduleFactory') it can be used directly as a module dependency (idem as uiRouter):

    import angular from 'angular';
    import uiRouter from 'angular-ui-router';
    import modelFactory from 'angular-model-factory';

    angular
        .module('myApp', [uiRouter, modelFactory])
        .controller('MainController', mainController)

**Comment by juristr** [comment]
@cosemansp Hi, I think this is something we're going to handle with #56 .
Meanwhile, we can add a main js file in `package.json` if that would make your life easier?

**Comment by cosemansp** [comment]
yes, if there is a main js file in the package.json I can include
'angular-model-factory' in my solution without any problem.

**Comment by juristr** [comment]
@cosemansp You could submit a PR :wink:. I'll merge it in then. Just make sure you quickly go over our contribution guidelines.

**Comment by amcdnl** [comment]
2.0 will be written in ES6.  WIP though.

**Comment by seb0zz** [comment]
Just made a pull request as this was getting important for me, too.
